lib/telemetry_metrics/monitor.ex

```diff
@@ -7,27 +7,53 @@ defmodule Membrane.TelemetryMetrics.Monitor do
 
   @spec start(:telemetry.event_name(), Membrane.TelemetryMetrics.label()) :: {:ok, pid()}
   def start(event_name, label) do
-    pid =
-      Process.spawn(
-        __MODULE__,
-        :run,
-        [self(), event_name, label],
-        []
-      )
-
-    {:ok, pid}
+    if :ets.whereis(Membrane.TelemetryMetrics.Monitor) == :undefined do
+      :ets.new(Membrane.TelemetryMetrics.Monitor, [:public, :set, :named_table])
```

Comment:
Create ETS table in the Application module and wrap name of the table in module attribute or access it using `__MODULE__`